### PR TITLE
Update to HIP-28 from Envision to catch-up with Guardian 2.0

### PR DESCRIPTION
**Pull request by anvabr**
**Description**:
Updating HIP-28 to bring up-to-date with Guardian 2.0. 
- updated architecture diagram
- updated wording on open source link available
- updated policy-workflow-state-objects table
- Added a new section for discoverability

**Related issue(s)**:
N/A

Fixes #
Minor fixes in formatting (removing white spaces etc)

**Notes for reviewer**:
N/A

**Checklist**

- [ X] Documented (Code comments, README, etc.)
- [X ] Tested (unit, integration, etc.)

### CODE REVIEW

File: HIP/hip-28.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 hip: 28
 title: Guardian Type Solution
-author: Matthew Smithies <matt.s@dovu.io>, Wes Geisenberger <wes.geisenberger@hedera.com>, Serg Metelin <sergey.metelin@hedera.com>, Ken Anderson <ken@hedera.com>, and Daniel Norkin <daniel.norkin@envisionblockchain.com>
+author: Matthew Smithies <matt.s@dovu.io>, Wes Geisenberger <wes.geisenberger@hedera.com>, Serg Metelin <sergey.metelin@hedera.com>, Ken Anderson <ken@hedera.com>, Daniel Norkin <daniel.norkin@envisionblockchain.com>, and Prernaa Agarwal <prernaa.agarwal@envisionblockchain.com>
```
Comment (sergmetelin) [inline HIP/hip-28.md:4]:
```suggestion
author: Matthew Smithies <matt.s@dovu.io>, Wes Geisenberger <wes.geisenberger@hedera.com>, Serg Metelin <sergey.metelin@hedera.com>, Ken Anderson <ken@hedera.com>, Daniel Norkin <daniel.norkin@envisionblockchain.com>, Prernaa Agarwal <prernaa.agarwal@envisionblockchain.com>
```

File: HIP/hip-28.md [hunk 2]
```diff
@@ -348,11 +363,15 @@ The following is the set of terms and definitions used to define Policy Actions
 |`children`|Defines a list of Blocks that are grouped into a Policy Workflow|List of Blocks, see also example given below|
 | `defaultActive` | Determines if Policy Workflow or a Policy Action can be executed | `true`                                           |
 | `permissions`   | Security Policy associated with a specific role                 | `ROOT_AUTHORITY`                                 |
+| `Roles`   | Available roles from which the user can choose             | `Installer`                                 |
```
Comment (sergmetelin) [inline HIP/hip-28.md:366]: Should these be capital letters (as in `Roles` vs `roles`, same for other changes in the table)

**Review by sergmetelin — CHANGES_REQUESTED** [review]

File: HIP/hip-28.md (hunk 2 above)
Comment (anvabr) [reply HIP/hip-28.md]: I think these need to be lower-case, will correct - thank you!

**Comment by mgarbs** [comment]
Will you fix the failing DCO check when you get a chance? https://github.com/hashgraph/hedera-improvement-proposal/pull/466/checks?check_run_id=6376355590

**Comment by anvabr** [comment]
> Will you fix the failing DCO check when you get a chance? https://github.com/hashgraph/hedera-improvement-proposal/pull/466/checks?check_run_id=6376355590

I've tried a few work-arounds, I am now not sure I can fix this. I don't have the permissions for 'Set DCO to pass', and going back and fixing every merge (not only mine) is not feasible. Can you please just use 'Set DCO to pass' and accept this PR? This is a guardian-specific PR which is what we develop.

**Comment by anvabr** [comment]
@sergmetelin the requested change has been made, please approve the PR?

**Pull request merged**